features/testbot/matching_turns.feature

```diff
@@ -1,105 +0,0 @@
-@routing @turns @testbot
```

Comment:
This testcase is identical to matching.feature, simply adding turn instructions.
Guidance should be tested on its own, not as a measure in other cases.